openapi/trireg2.yaml

```diff
@@ -2,7 +2,7 @@ openapi: "3.0.3"
 info:
   title: "TriReg2 API"
   description: "An OpenAPI specification for time registrations"
-  version: "0.0.7"
+  version: "0.0.8"
```

Comment:
Maybe change this to 0.1.0 instead, since we have some breaking changes in the API with this PR (regarding naming of input parameters etc)